### ISSUE TITLE
[EN][Raid] Blackwing Descent - Nefarian's End Heroic

### ISSUE THREAD

**Issue by Exertt** [Atlantiss/BugTracker#6409]
Animated Bone Warriors are bugged in Phase 1. They keep using their spell Hurl Bone, which is supposed to be used only when players are in unreachable position, e.g. on the pillars. They are just casting the spell randomly on the tank/kiter thus one shotting them. Making the fight almost impossible or rather pure RNG to see if you get randomly instant killed or not.

**Comment by ponpontos** [comment]
Hurl Bones is only supposed to be cast whenever someone is COMPLETELY out of range, like for example if you pull and there's someone up there outside the pit. 

They're not supposed to throw bones at just whoever has aggro on them inside the pit from range, that makes it impossible for a ranged to kite them. Using mages or hunters to kite them (or even frost DKs) was a common tactic, rendered completely useless here because you eventually just get nuked if you gain a bit of distance from them.

**Comment by Fapguy** [comment]
well that's great...

**Comment by linesar** [comment]
The ads can't really be kited in P3 anyway, cause they will outrun your hunter/mage and kill it anyway, so they have to be tanked no matter what, in normal it might work with a hunter or mage kiting them, but on heroic they would rape him as soon as they outrun his/her slows.

http://www.wowhead.com/npc=41918/animated-bone-warrior#comments
read the top comment here.

also, there seem to excist no guide of how "hurl bone" actually works, but http://us.battle.net/wow/en/forum/topic/6933235706 where the bug was that the mobs did not move, not that they cast it.

**Comment by ponpontos** [comment]
They can't be kited on P3 by a dps but they can on P1 until they run out of energy, making it easier for healers to heal the tanks on Onyxia and Nef. The top comment on that wowhead page is referring to kiting in P3, where you have the tank that isn't on Nef kiting them away from shadowblazes, making slows uncalled for. The general idea is that in P1 you have one tank on Onyxia, another pickes up Nefarian and the adds are dragged around and CCd until they run out of energy. Right now on the server this is impossible, you basically need to have one of the tanks tanking 1 dragon + the constructs or use a 3rd offtank for the constructs alone, making P3 harder. 

Also, there is no guide or even a  mention on how hurl bone actually works because the ability was basically never seen in a regular fight. It was only used in extremely unusual situations. They're not supposed to fucking stop chasing the target they have aggro on to cast hurl bone. It makes literally no sense.

**Comment by ponpontos** [comment]
This basically makes the fight a HUGE gearcheck until it gets fixed.

**Comment by linesar** [comment]
my message was meant to counter where you said "Hurl Bones is only supposed to be cast whenever someone is COMPLETELY out of range, like for example if you pull and there's someone up there outside the pit"
anyway, i found a clip here that might prove what you guys say. https://www.youtube.com/watch?v=EwCdqUzQIb8, if you look at this they have 1 add rooted, which does not prevent spell casting

**Comment by ponpontos** [comment]
There's a reason the guy is calling that a bug though. I'm assuming the problem might be related to pathing or something.

Also, you can link all the Nefarian videos you want and you will never find hurl bone being cast. It quite simply shouldn't be happening at all.

**Comment by linesar** [comment]
yeah i am watching as many as i can, and pretty much every video is using either onyxia tank or nefarian tank to tank the adds during P1

**Comment by Exertt** [comment]
I mean mainly P1 where it should be possible to have DK/Hunter to kite them until they die, which is unfortunately impossible on this server because they'll murder everyone with Hurl Bone.

I would guess it's just a problem with the pathing in that whole pit area.

**Comment by aneq** [comment]
"They're not supposed to fucking stop chasing the target they have aggro on to cast hurl bone. It makes literally no sense."
Hurl bone is heroic only spell, it does not appear on normal mode so kiting is a viable option only on normal mode.
And yes, they are supposed to cast hurl bone if you move out too far, if you keep them in ~10-15yd deadzone though, they will never cast hurl bone. This will get increasingly harder over time since as the stacks increase, the adds gain movement speed.
 I used to do this fight  as a p1 add tank on nef 25man heroic on this server and it's really possible to gather  all 12 of them coming from all sides without any of them casting bone toss unless some dps wants to "help" with stunning/slowing an add, so you outrange it and bone toss kills you. Kiting in p1 is also a pretty bad idea since they all have to die in a roughly same area, and you will be having a really hard time doing that with their increasing movespeed.
I'm assuming you're talking about 10man (since bringing an extra tank seems to be a problem for you, understandably). The way we did it on 10man hc was either plate dps tanking it, (this way we did 2 electrocutes in p1) or we just had onyxia tank tank the adds and burst onyxia down with bloodlust (0 electrocutes in p1). As you know, the add damage is pretty slow at first but it grows over time. Because of that you will need to chain CC on the adds and damage reduction cooldowns on whoever is tanking the adds.
People have done it on this server before (and before you start talking about people being overgeared, there was a guild that cleared all of t11 heroic within 60 days of levelling their characters to 85.)

**Comment by Exertt** [comment]
It is doable but that's not how it's supposed to work. Hurl bone shouldn't be ever casted unless you're out of the pit area.

**Comment by aneq** [comment]
Are you sure about that? The fight has a mechanic that kills people that are out of the pit - shadow of cowardice, it would make absolutely no sense to add another one.

**Comment by linesar** [comment]
http://us.battle.net/wow/en/forum/topic/6933235706 they casted it onr etail also.

**Comment by ponpontos** [comment]
aneq are you sure about what you're saying? There's literally no mention on the dungeon journal or any guide fight about the ability, which seems pretty odd for something that apparently has such an impact with the way it currently works on this server. There's no indication that the adds can't be kited (in fact, a good amount of retail guides and videos I saw used a kiting strategy in 10m) and wowhead doesn't list the ability as an heroic-only mechanic.

Also, from my experience, the distance amount is much closer to 10 than 15. The adds were at all times in range for my Howling Blast (15 yds without talent) and at a point they all stopped and casted hurl bone. I'm 100% sure I didn't move out of howling blast range and just got nuked for 300k physical damage just like that.

**Comment by linesar** [comment]
Ponpontus, can u plz link the guides that kite these skeletons. Cause all videos/guides i can find uses the nefarian or onyxia tank to tank the skeletons in P1.

**Comment by ponpontos** [comment]
http://web.archive.org/web/20110805235634/http://www.tankspot.com/showthread.php?74890-Heroic-Nefarian

Further down the page you can see someone talking about using a frost DK to kite the adds instead of using a 3rd tank as a viable strategy. I'll go through my youtube history to find a video when I have time.

**Comment by aneq** [comment]
Tankspot guides were made based on information from cataclysm beta and were valid for patch 4.0.3
the empower mechanic was different on 4.0.3 than on 4.0.6. On 4.0.3 adds gained empower stacks by autoattacking something (a tank for instance). On 4.0.6 the mechanic was changed and empower was gained over time. 
Since adds had no empower stacks, it was very easy to keep them in the "deadzone" because they didnt have any increased movspeed

**Comment by Exertt** [comment]
Yes I am sure, Hurl Bone is simply a mechanic to prevent any bugging of skeletons. There simply isn't any mention of Hurl Bone in any of the guides nor mentioned that you should be within a certain range of them to prevent them from casting it.

**Comment by Exertt** [comment]
One month without anything being done, lovely.

**Comment by Fapguy** [comment]
one month and 12 days :D

**Comment by Fapguy** [comment]
@Jenovas

**Comment by jangsna** [comment]
Fix this shit LMAO

**Comment by Jenovas** [comment]
eeeeeee no

**Comment by emouse** [comment]
We were trying to kill it last night, but people started getting randomly 1shot by the "Hurl Bone" ability.
Like we would have it hit our healers that are healing the Onyxia tank in P1 for absurd damage (around 1.6 million).
Any guide I watched and every post I read on the ability doesn't mention this issue, so I'm pretty sure it's still bugged.
If we try again tonight I will paste combat logs.

**Comment by emouse** [comment]
Here are some screenshots of couple of our pulls:
http://puu.sh/q5lms/def3270de0.jpg
http://puu.sh/q5lnM/1927e1809b.jpg

Random people just start getting hit by this, from all my reading up on it I can't make any sense of it.

**Comment by Fapguy** [comment]
Yeah emouse, hit them with more evidence... I quit playing on atlantiss because of all these bugs, and the complete lack of interest in even trying to fix them.

**Comment by emouse** [comment]
There is many more issues on this fight, but it's not as gamebreaking as the one with Hurl Bone.
- If you try to revive someone in the room there is a high chance they will just fall trough the floor
- Tail Lash will sometimes stun people that are nowhere near it, we had it hit me and the guy kiting the adds on basically the other side of the room
- Any "Blink type of ability" will put you under the textures (Shadowstep, Blink, Warlock Portal)
- You can abuse the Dominion mind control thing, if you get hit by the Tail Lash while in it you get control of you character but still have the 2 buttons, this means you can stack the buff to 150 easily, I was hitting the boss for 2 mil Pyroblast last night because of it.

**Comment by Exertt** [comment]
Bump
